roles/common/defaults/main.yml

```diff
@@ -17,6 +17,7 @@ common:
   serial_console:
     enabled: True
     name: ttyS0
+    baud_rate: 9600
```

Comment:
I don't know where this came from actually... This was not an intentional change. I'm going to revert this back to 115200 as it is in master. Good catch.